integrate/simpsons.go

```diff
@@ -27,31 +27,6 @@ func Simpsons(x, f []float64) float64 {
 		panic("integrate: must be sorted")
 	}
 
-	// Simpson's method approximates the integral of a function f by subdividing
```

Comment:
In the end I decided to delete this comment. It just repeats what's already in the code and moreover the code is probably clearer.